### PR TITLE
Enable appsec telemetry before waf init

### PR DESCRIPTION
**Pull request by iunanua**
### What does this PR do?
Enables appsec telemetry before waf init

### Motivation
`waf.init` metrics were missing

### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/library_config/nodejs.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

File: packages/dd-trace/src/appsec/index.js
Comment (simon-id) [inline packages/dd-trace/src/appsec/index.js:1]: Give me a test file then we can merge

**Review by simon-id — CHANGES_REQUESTED** [review]
Write a test then we can merge

File: packages/dd-trace/test/appsec/index.spec.js [hunk 1]
```diff
@@ -645,6 +646,61 @@ describe('AppSec Index', () => {
...
+        appsec: {
+          enabled: true
+        }
+      })
+    })
+
+    afterEach(() => {
+      appsec.disable()
+
+      appsecNamespace.reset()
```
Comment (uurien) [inline packages/dd-trace/test/appsec/index.spec.js:667]: I think that we should do it also `beforeEach`, can other non related test increment the `waf.init` metric and break these tests?

File: packages/dd-trace/test/appsec/index.spec.js [hunk 2]
```diff
@@ -645,6 +646,65 @@ describe('AppSec Index', () => {
...
+    beforeEach(() => {
+      sinon.restore()
+
+      appsecNamespace.reset()
+
+      config = new Config({
+        appsec: {
+          enabled: true
+        }
+      })
```
Comment (simon-id) [inline packages/dd-trace/test/appsec/index.spec.js:663]: that's already done line 38 no ?

Comment (iunanua) [reply packages/dd-trace/test/appsec/index.spec.js:663]: The waf doesn't validate the rules provided in line 38...

**Pull request merged**